mgwr/search.py

```diff
@@ -153,20 +153,19 @@ def equal_interval(l_bound, u_bound, interval, function, int_score=False):
     return opt_val, opt_score, output
 
 def multi_bw(init, y, X, n, k, family, tol, max_iter, rss_score,
-        gwr_func, bw_func, sel_func):
+        gwr_func, bw_func, sel_func, multi_bw_min, multi_bw_max):
     """
     Multiscale GWR bandwidth search procedure using iterative GAM backfitting
     """
-    if init:
+    if init is None:
         bw = sel_func(bw_func(y, X))
         optim_model = gwr_func(y, X, bw)
```

Comment:
Just one question. 
Wondering if the initial GWR optim_model takes account of the multi_bw_min/max when searching for its bandwidth. Say the user set the multi_bw_min/max to be [100] and [150] but the optimal bw for the initial GWR might be 50. Does it matter in this case?

Reply:
No, it doesn't. It will either find whatever bandwidth selection deems to be the optimal bandwidth or it will need to be set by the user using ``init_multi``. In either case, these are different parameterizations that do not interact. One is the bandwidth for the initial regression to produce the first round of partial residuals and the other is for the bandwidths of the GAM components after the initial regression. The value of the initial GWR BW could certainly affect the final outcome of the MGWR GAM routine, but does not seem to be very sensitive. Also, I anticipate (1) that most people will leave ``init_multi=None`` since this aligns with current best practices and (2) that the primary reason to use set the`` multi_bw_min`` and ``multi_bw_max`` would be to set them to the save value in which case the initial value doesn't seem to affect the end results. This was the use-case I introduced this functionality for, since we needed it for the simulations in the inference paper where the BW needs to be the same for each MGWR iteration.